### ISSUE TITLE
#430: build error - hanging test

### ISSUE THREAD

**Issue by uweschaefer** [factcast/factcast#431]
_From @uweschaefer on 2019-05-21 13:08:42 +0000 UTC_

_Copied from original issue: Mercateo/factcast#431_

**Comment by uweschaefer** [comment]
_From @codecov[bot] on 2019-05-21 13:08:58 +0000 UTC_

# [Codecov](https://codecov.io/gh/Mercateo/factcast/pull/431?src=pr&el=h1) Report
> Merging [#431](https://codecov.io/gh/Mercateo/factcast/pull/431?src=pr&el=desc) into [master](https://codecov.io/gh/Mercateo/factcast/commit/b979a0169cb1399f49eee72532a540c3f340f5b4?src=pr&el=desc) will **not change** coverage.
> The diff coverage is `n/a`.


```diff
@@           Coverage Diff           @@
##           master     #431   +/-   ##
=======================================
  Coverage   88.93%   88.93%           
=======================================
  Files          78       78           
  Lines        1870     1870           
  Branches      169      169           
=======================================
  Hits         1663     1663           
  Misses        126      126           
  Partials       81       81
```

**Comment by uweschaefer** [comment]
_From @uweschaefer on 2019-05-21 13:16:47 +0000 UTC_

could it be just a lack of heap? hopefully closes #430
